functions/Set-DbaMaxDop.ps1

```diff
@@ -138,7 +138,7 @@ function Set-DbaMaxDop {
                 Stop-Function -Message "Failure" -Category ConnectionError -ErrorRecord $_ -Target $servername -Continue
             }
 
-            if (!(Test-SqlSa -SqlInstance $server)) {
+            if (!(Test-SqlSa -SqlInstance $server -SqlCredential $SqlCredential)) {
```

Comment:
Thanks for the PR. Shouldn't $server be an already connected object that has already authenticated and thus does not need an extra sql cred?

Reply:
Yep, it should...but it does not work like that and the `Test-SqlSa` shows it has to see the credential object to test it properly.

Reply:
Just to note you can see how that function was used in the other commands and all of them also pass in the credential object. We can do it this way or work on fixing the internal function to properly validate the login connecting.

Reply:
okie dok, ty